PendingReleaseNotes

```diff
@@ -43,3 +43,15 @@
   * system
   * truncated
   * user_stats_sync
+
+>=17.2.1
+
+* The "BlueStore zero block detection" feature (first introduced to Quincy in
+https://github.com/ceph/ceph/pull/43337) has been turned off by default with a new global
+config option called `bluestore_zero_block_detection`. This feature, intended for large-scale
+synthetic testing, does not interact well with some RBD and CephFS components. Any side affects
+experienced in previous Quincy versions are now fixed, provided that the `bluestore_zero_block_detection`
```

Comment:
This isn't strictly true because if someone created a thick-provisioned image with 17.2.0 (that actually ended up being thin-provisioned due to zero block detecting kicking in), upgrading to 17.2.1 isn't going to fix that image:
```suggestion
experienced in previous Quincy versions would no longer occur, provided that the `bluestore_zero_block_detection`
```